File: Algorithms/Easy/1304_FindNUniqueIntegersSumUpToZero/Solution.cpp
class Solution {
public:
    vector<int> sumZero(int n) {
        vector<int> answer;
    
        if (n % 2 == 1)
            answer.push_back(0);
        
        n /= 2;
    
        for (int i = 1; i <= n; i++) {
            answer.push_back(i);
            answer.push_back(-i);
        }
    
        return answer;
    }
};

File: Algorithms/Easy/1304_FindNUniqueIntegersSumUpToZero/Solution.cs
public class Solution {
    public int[] SumZero(int n) {
        int[] answer = new int[n];
        int index = 0;
    
        if (n % 2 == 1)
            answer[index++] = 0;
        
        n /= 2;
        
        for (int i = 1; i <= n; i++) {
            answer[index++] = i;
            answer[index++] = -i;
        }
        
        return answer;
    }
}